src/QMCDrivers/WFOpt/QMCFixedSampleLinearOptimizeBatched.cpp

```diff
@@ -221,7 +224,7 @@ void QMCFixedSampleLinearOptimizeBatched::start()
   initialize_timer_.start();
   optTarget->getConfigurations("");
   optTarget->setRng(vmcEngine->getRngRefs());
-  optTarget->checkConfigurations();
+  optTarget->checkConfigurations(forCostFunc);
```

Comment:
hmmm the forCostFunction seems to just contain an unknown string and two nullptrs.  I remember tons of unchecked access through those two pointers above.